### ISSUE TITLE
Voice Access bugs

### ISSUE THREAD

**Issue by mklim** [flutter/flutter#40912]
/cc @amirh @goderbauer 

We appear to support basic navigation, but from some quick testing with the Gallery I'm seeing some issues.

- [ ] Text fields don't support input.
- [ ] Not all UI elements are correctly discovered and annotated. I'm not sure if this is an issue with the Gallery itself or an underlying semantics issue.

**Comment by xster** [comment]
flutter/gallery has some swipable elements that aren't swipable. I think there's some issues in the usage code since dev/integration_tests/flutter_gallery works ok.

**Comment by xster** [comment]
@chunhtai let's track issues here.

**Comment by xster** [comment]
Surprisingly showing keyboard, typing from soft keyboard, selecting text, moving cursor, copy/pasting all work. Just not typing. We're likely missing one callback in our Android embedding.
